packages/client/src/pages/admin/emojis.vue

```diff
@@ -134,13 +134,8 @@ const edit = (emoji) => {
 		emoji: emoji
 	}, {
 		done: result => {
-			if (result.updated) {
-				emojisPaginationComponent.value.replaceItem(item => item.id === emoji.id, {
-					...emoji,
-					...result.updated
-				});
-			} else if (result.deleted) {
-				emojisPaginationComponent.value.removeItem(item => item.id === emoji.id);
+			if (result.updated || result.deleted) {
+				emojisPaginationComponent.value.reload();
```

Comment:
The pagination component still has `updateItem` which is exported so I think at least that could be used here.

@syuilo I think it would be more performant to not reload the pagination component when a custom emoji gets deleted, but I am not sure why you removed the `removeItem` functionality here: https://github.com/misskey-dev/misskey/commit/586c11251a8c0e7ca9f8f3bbaad9bf745e6ef948#diff-412470ddca46a77339849fd70b56ce462cef9d5682a9186b505d5c6499f74b78

Reply:
I don't see any reason to remove it, so it would be better to re-implement `removeItem`.

Reply:
I reimplemented `removeItem` and adjusted the code for the emoji page to use that now! 👍